features/core-update.feature

```diff
@@ -209,7 +209,7 @@ Feature: Update WordPress core
     When I run `wp core update --minor`
     Then STDOUT should contain:
       """
-      Updating to version 4.0.17 (en_US)...
+      Updating to version 4.0.178 (en_US)...
```

Comment:
Updated

For the record, the test didn't fail because https://github.com/wp-cli/wp-cli/pull/4055 hasn't yet been applied to this repo.